airbyte-integrations/connectors/source-greenhouse/source_greenhouse/manifest.yaml

```diff
@@ -57,7 +57,6 @@ definitions:
         $ref: "#/definitions/requester"
   base_incremental_stream:
     $ref: "#/definitions/base_stream"
-    stream_cursor_field: "updated_at"
```

Comment:
Greenhouse is an example of a custom date time stream slicer where the cursor_field is an explicit field on the custom slicer. We can still extract it from there and store it on the stream